src/core/util/util.spec.js

```diff
@@ -443,6 +443,22 @@ describe('util', function() {
       flush(function(){ expect( callBackUsed ).toBeUndefined(); });
     }));
 
+    it('should only skip callbacks of scopes which were destroyed', inject(function($mdUtil) {
+      var callBackUsed1, callback1 = function(){ callBackUsed1 = true};
+      var callBackUsed2, callback2 = function(){ callBackUsed2 = true};
+      var scope1 = $rootScope.$new(true);
+      var scope2 = $rootScope.$new(true);
+
+      $mdUtil.nextTick(callback1, false, scope1);
```

Comment:
The old bug also had a problem with:

```javascript
$mdUtil.nextTick(callback1, false);  // no scope
$mdUtil.nextTick(callback2, false, scope);
$scope.$destroy();

flush(function() {
  expect(callback1).toHaveBeenCalled();
  expect(callback2).not.toHaveBeenCalled();
});
```

Would have failed previously.  Should I add that as a test too?